src/forms/dto/form-update.dto.ts

```
@@ -0,0 +1,61 @@
+    fields?: Record<string, unknown>;
+
+    @ApiProperty({
+        type: String,
+        description: 'tenantId',
+        example: 'Updated tenantId',
+    })
+    @IsString()
+    @IsOptional()
+    tenantId?: string;
+
+    @IsUUID()
```

ESLint

[error] Replace `····` with `··`

(prettier/prettier)